game-core/src/test/java/org/triplea/live/servers/ServerYamlParserTest.java

```diff
@@ -12,19 +12,19 @@
 import java.io.InputStream;
 import java.net.URI;
 import java.util.List;
-import java.util.function.Supplier;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
 import org.triplea.util.Version;
 
 class ServerYamlParserTest {
 
-  private static final Supplier<InputStream> SAMPLE_FILE_INPUT =
```

Comment:
The supplier was important before, I forget the exact reasoning why :(
I think the issue is that the read contents are cached. It was a deceptive situation previously where tests passed but they were reading stale input and a re-ordering of tests caused a failure. The other issue might have been input stream being re-read/exhausted. Again, can't really recall what the subtle point was (should have documented it), it's worth taking a double check here.

Reply:
Well I haven't converted it to a constant (because I assume that would've read the file too early potentially?), just made it a method so we should be fine